internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -19,56 +19,79 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"sigs.k8s.io/kubebuilder/v3/pkg/cli"
-	cfgv2 "sigs.k8s.io/kubebuilder/v3/pkg/config/v2"
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+	golangv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v2"
+	golangv3 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v3"
 
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/bundle"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/cleanup"
-	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/completion"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/generate"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/olm"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/run"
 	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/scorecard"
 	"github.com/operator-framework/operator-sdk/internal/flags"
+	"github.com/operator-framework/operator-sdk/internal/plugins"
 	ansiblev1 "github.com/operator-framework/operator-sdk/internal/plugins/ansible/v1"
-	golangv2 "github.com/operator-framework/operator-sdk/internal/plugins/golang/v2"
-	golangv3 "github.com/operator-framework/operator-sdk/internal/plugins/golang/v3"
+	envtestv1 "github.com/operator-framework/operator-sdk/internal/plugins/envtest/v1"
 	helmv1 "github.com/operator-framework/operator-sdk/internal/plugins/helm/v1"
+	manifestsv2 "github.com/operator-framework/operator-sdk/internal/plugins/manifests/v2"
+	scorecardv2 "github.com/operator-framework/operator-sdk/internal/plugins/scorecard/v2"
 	"github.com/operator-framework/operator-sdk/internal/util/projutil"
 )
 
 var commands = []*cobra.Command{
 	bundle.NewCmd(),
 	cleanup.NewCmd(),
-	completion.NewCmd(),
 	generate.NewCmd(),
 	olm.NewCmd(),
 	run.NewCmd(),
 	scorecard.NewCmd(),
 }
 
 func Run() error {
-	cli, _ := GetPluginsCLIAndRoot()
-	return cli.Run()
+	c, _ := GetPluginsCLIAndRoot()
+	return c.Run()
 }
 
 // GetPluginsCLIAndRoot returns the plugins based CLI configured to use operator-sdk as the root command
 // This CLI can run kubebuilder commands and certain SDK specific commands that are aligned for
 // the kubebuilder project layout
 func GetPluginsCLIAndRoot() (cli.CLI, *cobra.Command) {
+	ansibleBundle, _ := plugin.NewBundle("ansible"+plugins.DefaultNameQualifier, plugin.Version{Number: 1},
+		ansiblev1.Plugin{},
+		manifestsv2.Plugin{},
+		scorecardv2.Plugin{},
+	)
+	gov2Bundle, _ := plugin.NewBundle(golangv2.Plugin{}.Name(), golangv2.Plugin{}.Version(),
+		golangv2.Plugin{},
+		envtestv1.Plugin{},
+		manifestsv2.Plugin{},
+		scorecardv2.Plugin{},
```

Comment:
Anyway, I also see a great value for the BundlePlugin idea in the future where we might want to create a bundle to address the requirement to provide hybrid operators such as; go+helm+caveats. So, I am ok with this solution to ensure backwards compatibilities and bring the flexibilities for we start to address these needs. 

However, as we spoke conceptually, we cannot address, for example, the go scenario as a new plugin such as `go.sdk.operatorframework.io` because it is not the result/behaviour that we would like for it. The goal is SDK scaffolding the same GO Kubebuilder plugins and runs its customizations on top. more info:  https://github.com/operator-framework/operator-sdk/pull/4581#discussion_r584936851 

**So,  imo**
- shows fine we use the Bundle for now
- I think that the bundle idea will have its purpose and proper requirements in the future
- but, before getting it all merged we need to have an issue tracked into upstream with the requirement described for we are able to address it properly in the future. Also, I'd like to see in the SDK code/side a comment with todo linking the issue.